src/ctia/task/migration/store.clj

```diff
@@ -55,7 +55,8 @@
 
 (def timeout (* 5 60000))
 (def es-max-retry 3)
-(def migration-es-conn (atom nil))
+;;FIXME refactor to local argument
```

Comment:
I created this issue for me and added it to the ES7 migration milestone.
https://github.com/threatgrid/iroh/issues/4230

Reply:
Thanks!